src/get_hashed_strings.js

```diff
@@ -0,0 +1,96 @@
+/*
+Script to get the translation string hashes for a given resource from Transifex.
+
+Sample usage: node src/get_hashed_strings.js --resource=studio-frontend --token=<transifex_api_token> --outputJsonDirectory=bash_scripts
+
+When running on local, outputJsonDirectory should be overwritten to avoid DirectoryNotFound error
```

Comment:
What does "outputJsonDirectory should be overwritten" mean?

Reply:
I will update this comment. What I meant was that the default output directory is `.node_modules/reactifex/bash_scripts` for whenever the script would be run by any MFE. On local testing inside reactifex itself, this directory will not be found because reactifex is not installed in reactifex.